### PR TITLE
css cards styling

### PR DESCRIPTION
**Pull request by tristan0508**
Need approval

### CODE REVIEW

**Comment by ste163** [comment]
Found an issue! The cards do not resize based on screen size.

File: css/main.css [hunk 1]
```diff
@@ -10,8 +39,114 @@ section{
 }
 
 
-.cards {
-    border: 5px solid black;
+.cities {
+    min-width: 350px;
```
Comment (ste163) [inline css/main.css:43]: I think it's here where we need to have something with em or a percentage to get the resizing right.

Comment (ste163) [reply css/main.css:43]: Or maybe it's actually a flexbox thing...

**Comment by AudreyThomasson** [comment]
need to look at the cards

**Comment by ste163** [comment]
Checked and looks good!

File: css/main.css (hunk 1 above)
Comment (marceljones) [reply css/main.css:43]: Way to go!

**Review by AudreyThomasson — APPROVED** [review]
Yes

**Review by ste163 — APPROVED** [review]
Looks good

**Review by marceljones — APPROVED** [review]
Small potatoes!

**Pull request merged**